base/loading.jl

```diff
@@ -746,6 +753,7 @@ function _require_search_from_serialized(pkg::PkgId, sourcepath::String)
         if staledeps === true
             continue
         end
+        touch(path_to_try) # Update timestamp of precompilation file
```

Comment:
The timestamps of the .ji files themselves have no relevance of invalidating stuff right?